servicex/query_core.py

```diff
@@ -285,7 +285,25 @@ def transform_complete(task: Task):
         )
 
         if not cached_record:
-            self.request_id = await self.servicex.submit_transform(sx_request)
+            ###
+            # if the transform request is pending 
```

Comment:
Please don't check in commented out code. If we need this code it is in the commit history